osu.Game.Modes.Osu/Objects/Slider.cs

```diff
@@ -53,14 +53,10 @@ public override void SetDefaultsFromBeatmap(Beatmap beatmap)
             base.SetDefaultsFromBeatmap(beatmap);
 
             var baseDifficulty = beatmap.BeatmapInfo.BaseDifficulty;
+            var baseVelocity = 100 * baseDifficulty.SliderMultiplier;
 
-            ControlPoint overridePoint;
-            ControlPoint timingPoint = beatmap.TimingPointAt(StartTime, out overridePoint);
-            var velocityAdjustment = overridePoint?.VelocityAdjustment ?? 1;
-            var baseVelocity = 100 * baseDifficulty.SliderMultiplier / velocityAdjustment;
-
-            Velocity = baseVelocity / timingPoint.BeatLength;
-            TickDistance = baseVelocity / baseDifficulty.SliderTickRate;
+            Velocity = baseVelocity / beatmap.BeatLengthAt(StartTime);
+            TickDistance = baseVelocity / baseDifficulty.SliderTickRate / beatmap.BPMMultiplierAt(StartTime);

```

Comment:
Don't do double divides. Order of execution is confusing in such cases. Put parentheses or transform it into `bla / (bla2 * bla3)`